markdown-format/README.md

```diff
@@ -59,6 +88,69 @@ It has multiple sentences.
 Let's format it!
 ```
 
+## Integration with Formatting Tools
+
+See the [examples/](examples/) directory for complete configuration files and sample markdown files demonstrating the integrations.
+
+### treefmt
+
+[treefmt](https://github.com/numtide/treefmt) is a universal code formatter that runs multiple formatters with one command.
+
+Add to your `treefmt.toml`:
+
+```toml
+[formatter.markdown-format]
+command = "markdown-format"
+options = ["-w"]
+includes = ["*.md"]
+```
+
+Then run:
+
+```bash
+treefmt
+```
+
+### dprint
+
+[dprint](https://dprint.dev/) is a pluggable and configurable code formatter.
+
+Since markdown-format uses a custom sentence-per-line format, you'll need to use dprint's process plugin or create a custom wrapper. Here's how to integrate using a process-based approach:
+
+1. Create a wrapper script that handles dprint's stdin/stdout protocol (e.g., `dprint-markdown-format.sh`):
+
+```bash
+#!/bin/bash
+# Wrapper script for dprint integration
+
+# Read from stdin, format, and output to stdout
+/path/to/markdown-format -
+```
+
+2. Add to your `dprint.json`:
+
+```json
+{
+  "plugins": [
+    "https://plugins.dprint.dev/exec-0.5.0.json@checksum"
```

Comment:
adding the exec plugin should be handled with `dprint add` or smth, check the dprint docs